tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -3948,4 +3948,172 @@ TestCarePartnerParticipantInfoEditOrAddEmergencyRoomAdmissionsInfo:
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_records.demographics_tab$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_records.demographics_vendor_subtab$
+  # Variables to use for Add Vendor
+      Vars:
+        ADDRESS: 1000 Street Name, Somewhere, GA 30024
+        WEBSITE: www.airportridesrus.com
+        CATEGORY: careCoordination
+        PHONE: 4365565897
+    - Type: generate_unique_name
+      Name: 'Airport Rides "R" Us'
+      ResultVar: NAME
+  # Steps to add a vendor
+    # Clicking on "Add Vendor" button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.add_vendor_button$
+    # Waiting for modal
+    - Type: wait_for
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.vendor_modal$
+    # Closing "Add Vendor" modal
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.cancel_vendor_modal_button$
+   # Wait to close modal
+    - Type: wait_not_visible
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.modal$
+    # Open Add "Vendor" modal
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.add_vendor_button$
+  # Validate if "Add vendor" is opened
+    - Type: get_text
+      Strategy: xpath
+      Id : $PAGE.care_platform_edit_participants_vendor.modal_title_label$
+      Greps:
+        - var: MODAL_TITLE
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: eq
+          Var: MODAL_TITLE
+          Value: Add Vendor
+  # Filling Vendor form 
+    - Type: click
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.select_category_modal_input$
+    # Selecting a Category
+    - Type:  wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.category_modal_input$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.category_modal_input$
+    # Filling input form
+    - Type: send_keys
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.field_name_modal_input$
+      Value: $AND_CLI_NAME$
+    - Type: send_keys
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.field_address_modal_input$
+      Value: $AND_CLI_ADDRESS$
+    - Type: send_keys
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.field_phone_modal_input$
+      Value: $AND_CLI_PHONE$
+    - Type: send_keys
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.field_website_modal_input$
+      Value: $AND_CLI_WEBSITE$
+  # Click on "save and close" button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.save_close_modal_button$
+  # Validate tostify message and wait until toast is not visible
```

Comment:
Can we change this to be toast message instead of tostify and update it everywhere it is mentioned?

Reply:
I applied the changes in this commit https://github.com/neveralone-chs-org/TestRay/commit/48ed04279808ef48bc2b94fa082f7029b67617a3